spec/relaton_ccsds/bibliography_spec.rb

```diff
@@ -1,17 +1,36 @@
-describe RelatonCcsds::Bibliography do
+class RelatonCcsds::TestHitCollection < RelatonCcsds::HitCollection
+  # override default index method to avoid index downloading
+  def index
+    @index ||= Relaton::Index.find_or_create :ccsds, file: "index-v2.yaml"
+  end
+
+  # method to be able to add index rows from test's context
+  def add_to_index(id, file)
+    index.add_or_update(id, file)
+  end
+end
+
+module RelatonCcsds::TestBibliography
+  include RelatonCcsds::Bibliography
+  extend self
+  def search(ref)
+    hit_collection = RelatonCcsds::TestHitCollection.new(ref)
+    hit_collection.add_to_index(Pubid::Ccsds::Identifier.parse("CCSDS 230.2-G-1"), "data/CCSDS-230-2-G-1.yaml")
+    hit_collection.add_to_index(Pubid::Ccsds::Identifier.parse("CCSDS 720.4-Y-1"), "data/CCSDS-720-4-Y-1.yaml")
+    hit_collection.add_to_index(Pubid::Ccsds::Identifier.parse("CCSDS 650.0-M-2"), "data/CCSDS-650-0-M-2.yaml")
+    hit_collection.add_to_index(Pubid::Ccsds::Identifier.parse("CCSDS 650.0-M-2 - French Translated"), "data/CCSDS-650-0-M-2-French-Translated.yaml")
+
+    hit_collection.fetch
+  end
+end
+
+describe RelatonCcsds::TestBibliography do
   before do
     # Force to download index file
     allow_any_instance_of(Relaton::Index::Type).to receive(:actual?).and_return(false)
     allow_any_instance_of(Relaton::Index::FileIO).to receive(:check_file).and_return(nil)
   end
 
-  it ".searche" do
```

Comment:
@andrew2net the test start crashing after changes I made, then I check the code and found:
```
    def search(ref)
      RelatonCcsds::HitCollection.new(ref).fetch
    end
```
The test for HitCollection#fetch we already have. Cases like this I usually just skip if it's one-liner like this where it's pretty hard to make a mistake.
But I don't know, maybe there are better approach.

Reply:
Ok, but now this method isn't tested at all because it's replaced in the `TestBibliography` module
<img width="515" alt="image" src="https://github.com/user-attachments/assets/6d1147c0-d2f7-440b-879f-c8bfce07b0fb">